content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -68,17 +68,20 @@ If only certain stages should be used to track days of inactivity, they can be a
    - `Customer Feedback`
    - `Closed`
 
-   Tickets can linger in the :guilabel:`Customer Feedback stage`, because once an issue is solved,
+   Tickets may linger in the :guilabel:`Customer Feedback` stage, because once an issue is solved,
    customers may not respond immediately. At that point, the tickets can be closed automatically.
-   However, tickets in the :guilabel:`New` and :guilabel:`In Progress` stages may remain inactive
-   due to assignment or workload issues. Closing these tickets automatically would result in issues
+
+   Tickets in the :guilabel:`New` and :guilabel:`In Progress` stages could remain inactive due to
+   assignment or workload issues. The support team may be looking into the issue even if they are
+   not updating the ticket directly. Closing these tickets automatically would result in issues
    going unsolved.
 
-   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below\:\
+   Therefore, the :guilabel:`Automatic Closing` settings for this team would be configured as
+   below\:\
```

Comment:
is there a reason behind the slashes before and after the ":" here -- it looks like it renders fine when I run 'make fast.' I just wasn't super sure if this was intentional or not -- either way, just figured it was worth bringing to your attention. 😄